lib/rspec/openapi.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'bundler/setup'
```

Comment:
Is `bundler` gem always included in runtime?
If not, it should be added to `add_dependency` section:
https://github.com/exoego/rspec-openapi/blob/cdda8fc479618f3c44728f84c67eeb938df99aef/rspec-openapi.gemspec#L31-L33



Since `bundler` is not so small (400KB+ since 2.4.3), it would be preferable not to include extra dependency by using Ruby's native mechanism like `defined?(Rails)` or `defined?(Hanami)`.

Reply:
I tried not using bundler and using the standard methods you suggested, however at this point it seems due to the loading order, these constants don't exist yet.
If i try this:
```ruby
module SharedHooks
  def self.find_extractor
    if defined?(Rails) && Rails.respond_to?(:application) && Rails.application
      require 'rspec/openapi/extractors/rails'

      RSpec::OpenAPI::Extractors::Rails
    elsif defined?(Hanami) && Hanami.respond_to?(:app) && Hanami.app?
      require 'rspec/openapi/extractors/hanami'

      RSpec::OpenAPI::Extractors::Hanami
      # elsif defined?(Roda)
      #   some Roda extractor
    else
      require 'rspec/openapi/extractors/rack'

      RSpec::OpenAPI::Extractors::Rack
    end
  end
end
```
it won't work for hanami because of the way I use to throw the inspector into the router.

Would be glad for help in this aspect

Reply:
How about 

```ruby
begin
   require 'hanami'
   require 'rspec/openapi/extractors/hanami'
rescue
   puts 'Hanami not detected'
end

begin
   require 'rails'
   require 'rspec/openapi/extractors/rails'
rescue
   puts 'Rails not detected'
end
```